app/services/patient_deduplication/runner.rb

```diff
@@ -31,6 +31,10 @@ def handle_error(e, patient_ids)
     def report_summary
       Rails.logger.info(report_stats.to_json)
       Rails.logger.info "Failed to merge patients #{merge_failures}"
+
+      Statsd.instance.count("PatientDeduplication.total_processed", duplicate_patient_ids.flatten.count)
+      Statsd.instance.count("PatientDeduplication.total_merged", duplicate_patient_ids.flatten.count - merge_failures.flatten.count)
+      Statsd.instance.count("PatientDeduplication.total_failures", merge_failures.flatten.count)
```

Comment:
Can we add this into the dashboard controller as well? Perhaps we can use a tag to differentiate automatic vs manual while keeping the metric names the same? https://docs.datadoghq.com/developers/metrics/dogstatsd_metrics_submission/#metric-tagging

Reply:
Addressed in adcac5155eac57195301b3094c36a21b1c0f1c96.